webapp/app/components/listProgress/listProgress.component.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { LinearProgress } from 'material-ui/Progress';
+
+
+export class ListsProgress extends PureComponent {
+  static propTypes = {
+    list: PropTypes.object,
+  };
+
+  state = {
+    value: 0,
+  };
+
+  componentDidMount() {
+    this._timeout = setTimeout(() => this.setState({ value: this.getProgressValue(this.props.list) }));
+  }
+
+  componentDidUpdate() {
+    this.setState({ value: this.getProgressValue(this.props.list) });
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this._timeout);
+  }
+
+  getLeftItemsCount = (list) => list.get('items').filter((item) => item.get('active')).size;
```

Comment:
you don;t need to pass the list to this function, you can take the value directly from props